src/adapter/graphql/tests/tests/test_gql_dataset_flow_runs.rs

```diff
@@ -80,6 +89,66 @@ async fn test_trigger_ingest_root_dataset() {
             }
         })
     );
+
+    let request_code = FlowRunsHarness::list_flows_query(&create_result.dataset_handle.id);
+    let response = schema
+        .execute(
+            async_graphql::Request::new(request_code.clone())
+                .data(harness.catalog_authorized.clone()),
+        )
+        .await;
+
+    assert!(response.is_ok(), "{:?}", response);
+    assert_eq!(
+        response.data,
+        value!({
+            "datasets": {
+                "byId": {
+                    "flows": {
+                        "runs": {
+                            "listFlows": {
+                                "nodes": [
+                                    {
+                                        "flowId": "0",
+                                        "description": {
+                                            "__typename": "FlowDescriptionDatasetPollingIngest",
+                                            "datasetId": create_result.dataset_handle.id.to_string(),
+                                            "ingestedRecordsCount": Value::Null,
```

Comment:
FYI, I believe this macro also accepts just `null` instead of `Value::Null`

Reply:
Weird, but worked. There is nothing about it in the docs, but it's a macro-specific construct.

```
    // Next element is `null`.
    (@array [$($elems:expr,)*] null $($rest:tt)*) => {
        $crate::value_internal!(@array [$($elems,)* $crate::value_internal!(null)] $($rest)*)
    };
```